### PR TITLE
feature(gemini): update gemini to 1.8.2

### PR DESCRIPTION
**Pull request by soyacz**
Recently many changes were added to Gemini.

Update default Gemini version to 1.8.2 and fix releasing process as
docker images are now uploaded automatically upon releasing Gemini.
Because new docker image changed entrypoint and dir structure, some
adaptations were required to the way we run it.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fruch — APPROVED** [review]
LGTM

If the Gemini integration test pass, we can merge

**Comment by aleksbykov** [comment]
Lgtm

**Review by aleksbykov — APPROVED** [review]

File: defaults/test_default.yaml [hunk 1]
```diff
@@ -205,7 +205,7 @@ stress_image:
   nosqlbench: 'scylladb/hydra-loaders:nosqlbench-4.15.49'
   cassandra-stress: '' # default would be same version as scylla under test
   scylla-bench: 'scylladb/hydra-loaders:scylla-bench-v0.1.17'
-  gemini: 'scylladb/hydra-loaders:gemini-1.8.0'
+  gemini: 'scylladb/hydra-loaders:gemini-1.8.2'
```
Comment (vponomaryov) [inline defaults/test_default.yaml:208]: The built image has `v` in the tag name:
https://hub.docker.com/layers/scylladb/hydra-loaders/gemini-v1.8.2/images/sha256-f8f566f0c5845d6fabc517c29f72fbcf996e57feb546f598bd05355aec9ca910?context=explore

And yes, it is different than it was before without `v` -> https://hub.docker.com/layers/scylladb/hydra-loaders/gemini-1.8.0/images/sha256-da5b5b4b46c1e7da9ab7899b1e0c228d4be2d4923cd53380ec34b37f26a8d943?context=explore

So, need to fix one of the 2 places.

**Review by vponomaryov — CHANGES_REQUESTED** [review]

File: docker/gemini/image [hunk 2]
```diff
@@ -1 +1 @@
-scylladb/hydra-loaders:gemini-1.8.0
+scylladb/hydra-loaders:gemini-v1.8.0
```
Comment (vponomaryov) [inline docker/gemini/image:1]: It has typo, must be `gemini-v1.8.2`

**Review by vponomaryov — CHANGES_REQUESTED** [review]

File: docker/gemini/image (hunk 2 above)
Comment (fruch) [reply docker/gemini/image:1]: this file should be remove, nothing uses it.

Comment (soyacz) [reply docker/gemini/image:1]: I changed it by mistake, but it has no meaning anyway.

File: defaults/test_default.yaml (hunk 1 above)
Comment (soyacz) [reply defaults/test_default.yaml]: there were more changes besides naming... I adapted SCT.

**Review by vponomaryov — APPROVED** [review]
LGTM

**Comment by aleksbykov** [comment]
@soyacz @fruch , could you backport this PR to 2023.1 branch?

**Comment by fruch** [comment]
> @soyacz @fruch , could you backport this PR to 2023.1 branch?

There are too many issues with it on master duty runs, I don't think it would be wise.

I'm waiting for @dkropachev response, to see if he found the root cause of those failures

**Pull request merged**